examples/vcnl4010_simpletest.py

```diff
@@ -1,3 +1,6 @@
+# SPDX-FileCopyrightText: 2021 ladyada for Adafruit Industries
+# SPDX-License-Identifier: MIT
+
 # Simple demo of the VCNL4010 proximity and light sensor.
 # Will print the proximity and ambient light every second.
 # Author: Tony DiCola
```

Comment:
This one has a different author listed here

Reply:
Latest commit changes author to Tony, and I changed the date based on the initial commit that I found in github history.